gl_autotool/tests/base_test.py

```diff
@@ -3,12 +3,20 @@
 
 
 class BaseTest:
+    BASE_URL = "https://ultimateqa.com/automation"
+    DEFAULT_BROWSER = "chromium"
```

Comment:
In our project, all class attributes are only lowercase. Upper case we use for either top-level module constants (outside the class) or in Enum classes.

```suggestion
    base_url= "https://ultimateqa.com/automation"
    default_browser = "chromium"
```